src/main/java/Main.java

```diff
@@ -158,6 +178,23 @@ public static void main(String... args) {
 
         registerShutdownHook(graphDb);
 
+        if (enableNeo4jWebServer) {
+            // Start Neo4jWebserver
+            // Deprecated but useful for debugging and exploring the database
+            LOGGER.info("Starting Neo4j WebServer on 127.0.0.1 Port: " + configNeo4jWebPort);
+            WrappingNeoServerBootstrapper neoServerBootstrapper;
+            GraphDatabaseAPI api = (GraphDatabaseAPI) graphDb;
+
+            ServerConfigurator config = new ServerConfigurator(api);
+            config.configuration()
+                    .addProperty(Configurator.WEBSERVER_ADDRESS_PROPERTY_KEY, "127.0.0.1");
```

Comment:
I don't know, I did it because by default the database has no password protection.
But we could change it and print a warning that a password should be set.
I would suggest we change it when someone needs it.